src/vim-loader/scene.ts

```diff
@@ -56,34 +57,12 @@ export class Scene {
   /**
    * Sets vim index for this scene and all its THREE.Meshes.
    */
-  setIndex (index: number) {
+  setVim (vim: Vim) {
```

Comment:
Minor: I think we should update the documentation above this function since it is no longer referring to an index